myhpom/migrations/0016_scribbles_data.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.core.urlresolvers import reverse
+from scribbler.models import Scribble
+
+
+SCRIBBLES = [
+    {'slug': 'signup-tos-short', 'url': reverse('myhpom:signup'), 'content': '{% lorem 32 w %}'},
```

Comment:
Starting with a fresh database dump (`./hsctl rebuild --db`), I'm having trouble with this `reverse()` here - it pulls in sufficient database relations that are expected to exist, ultimately bailing with a stack trace.

Though it pains me to say it, it would probably be better to use a hard coded value here.

On a related note I'm unable to run tests (and I notice that CI hasn't succeeded) - I get a weird migration exception for the example project in scribbler:

```
    cursor.execute(sql, params)
  File "/usr/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python2.7/site-packages/django/db/utils.py", line 98, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: relation "scribbler_dayslog" does not exist
LINE 1: ...slog"."id", "scribbler_dayslog"."happenings" FROM "scribbler...
```

Appears to be from here: https://github.com/caktus/django-scribbler/blob/master/example/dayslog/migrations/0001_initial.py

Which is weird b/c...I wouldn't think that migration should be detected or run at all!

Reply:
So I was using reverse(‘myhpom:signup’) in the migrations/0016_scribbles_data.py, and that is evidently a no-no. I’ll have to hard-code that URL as well. I’m also making it ‘shared’.